frontend/src/metabase/entities/containers/rtk-query/types.ts

```diff
@@ -0,0 +1,61 @@
+import type { UseQuery } from "@reduxjs/toolkit/dist/query/react/buildHooks";
```

Comment:
Entity framework internals meet TypeScript for the first time.
Beware: types in this file are incomplete and may be inaccurate.